### PR TITLE
Feat/avatar

### PR DESCRIPTION
**Pull request by stephankaag**

### CODE REVIEW

File: src/prefabs/avatar.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+(() => ({
+  name: 'Avatar',
+  icon: 'ImageInputIcon',
```
Comment (vdberganton) [inline src/prefabs/avatar.js:3]: This is not the correct icon for in the sidebar

File: src/components/avatar.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+(() => ({
+  name: 'Avatar',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: ['CONTENT_COMPONENT'],
+  orientation: 'HORIZONTAL',
```
Comment (vdberganton) [inline src/components/avatar.js:5]: I expect this to be a VERTICAL orientation

File: src/components/avatar.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        {children}
+      </Avatar>
+    );
+
+    return isDev ? <div>{AvatarComponent}</div> : AvatarComponent;
+  })(),
+  styles: () => () => ({
+    avatar: {
+      width: ({ options: { width } }) => width || 'auto',
+      height: ({ options: { height } }) => height || 'auto',
```
Comment (vdberganton) [inline src/components/avatar.js:28]: The height option does not appear to work in DEV mode

File: src/components/avatar.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        variant={variant}
+        alt={imgAlt}
+        src={imgUrl}
+        className={classes.avatar}
+      >
+        {children}
+      </Avatar>
+    );
+
+    return isDev ? <div>{AvatarComponent}</div> : AvatarComponent;
```
Comment (vdberganton) [inline src/components/avatar.js:23]: Missing a placeholder

Comment (stephankaag) [reply src/components/avatar.js:23]: Schermafbeelding 2020-05-22 om 13 05 37

This looks pretty placeholdery to me? What kind of placeholder do you expect?

Comment (vdberganton) [reply src/components/avatar.js:23]: Same as the image component

**Comment by vdberganton** [comment]
Closing this PR because we added our own Avatar component.

**Pull request closed without merging**